### PR TITLE
[native][Please do not review] Run presto-tests with prestissimo

### LINKED ISSUE #19974: [Native] Run presto-tests module with Prestissimo to establish e2e query runs with presto build
presto-tests (and presto-product-tests) is the most comprehensive e2e test suite for Presto Java. Running it with Prestissimo establishes a good baseline for presto_cpp quality with each build.

Add support to run these tests with HiveExternalWorkerQueryRunner in addition to the Java Query Runner.

### PR DESCRIPTION
**Pull request by pramodsatya**
Testing an approach to resolve #19974 

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-native-execution/src/test/java/com/facebook/presto/nativetests/TestNativeOrderByQueries.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    }
+
+    @Override
+    protected ExpectedQueryRunner createExpectedQueryRunner() throws Exception
+    {
+        return PrestoNativeQueryRunnerUtils.createJavaQueryRunner();
+    }
+
+    @Override
+    @Test
```
Comment (aditi-pandit) [inline presto-native-execution/src/test/java/com/facebook/presto/nativetests/TestNativeOrderByQueries.java:39]: Can you disable testing and check if that works ?

File: presto-native-execution/src/test/java/com/facebook/presto/nativetests/TestNativeOrderByQueries.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    public void testOrderByWithOutputColumnReference()
```
Comment (aditi-pandit) [inline presto-native-execution/src/test/java/com/facebook/presto/nativetests/TestNativeOrderByQueries.java:40]: i) Lets not rewrite SQL -- use assertQueryFails with the error message and we fix them over time.

Comment (pramodsatya) [reply presto-native-execution/src/test/java/com/facebook/presto/nativetests/TestNativeOrderByQueries.java]: Thanks for the suggestion, the assertQuery rewrites are reverted. assertQueryFails expects the Sql query and the error message as the parameters. The failing queries all have expectedResult (in Sql) as a parameter, which cannot be added to assertQueryFails. 
The failing tests have been moved to a separate disabled testcase. Could you please confirm if this is fine?

File: presto-native-execution/src/test/java/com/facebook/presto/nativetests/TestPrestoNativeOrderByQueries.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        assertQuery(
+                "WITH t AS (SELECT 1 x, 2 y) SELECT x, y FROM t ORDER BY x, y LIMIT 1",
+                "SELECT 1, 2");
+        assertQuery(
+                "WITH t AS (SELECT 1 x, 1 y) SELECT x, y FROM t ORDER BY x, y LIMIT 1",
+                "SELECT 1, 1");
+    }
+
+    @Test(enabled = false)
+    public void testOrderByFailingQueries()
```
Comment (mknegi) [inline presto-native-execution/src/test/java/com/facebook/presto/nativetests/TestPrestoNativeOrderByQueries.java:110]: Was it necessary to create this another method of keep track of all the failing queries ? For example whenever the apply scalar function is implemented in velox then we could have simply removed this testOrderByWithOutputColumnReference() method  from this class and relied on common definition in AbstractTestOrderByQueries class and maintain as much common code as possible going forward.

**Pull request closed without merging**